worker/package.json

```diff
@@ -5,10 +5,14 @@
 	"license": "MIT",
 	"dependencies": {
 		"mime": "^2.4.4",
-		"path-to-regexp": "^3.1.0"
+		"path-to-regexp": "^3.1.0",
+		"buffer": "*",
+		"@sagi.io/workers-jwt": "*"
 	},
 	"devDependencies": {
-		"bili": "^4.8.1"
+		"bili": "^4.8.1",
+		"rollup-plugin-node-builtins":"*",
```

Comment:
Please don't use "*" as version number.
Just use what appears after running `npm install -S $pkg` or `yarn install $pkg`

Reply:
Others packages using "*" should be fixed as well. As long as this is fixed, I can merge that PR now.

Reply:
I've pushed the fix